spec/exports-spec.ts

```diff
@@ -19,7 +19,7 @@ import { onErrorResumeNext } from '../src/internal/observable/onErrorResumeNext'
 import { pairs } from '../src/internal/observable/pairs';
 import { race } from '../src/internal/observable/race';
 import { range } from '../src/internal/observable/range';
-import { _throw } from '../src/internal/observable/throw';
+import { throwError } from '../src/internal/observable/throwError';
```

Comment:
was just `error` considered?

Reply:
@jayphelps ... yes, but the roundness of `throwError` and `catchError` seemed to make sense. But we're in alpha, so it's definitely open to suggestion.